derp/derp_server.go

```diff
@@ -1955,6 +1955,22 @@ func (s *Server) ConsistencyCheck() error {
 			s.curClients.Value(),
 			len(s.clients)))
 	}
+	if s.verifyClientsLocalTailscaled {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+		defer cancel()
+		status, err := localClient.Status(ctx)
+		if err != nil {
+			errs = append(errs, fmt.Sprintf("localClient.Status: %v", err))
+		}
+		info := &clientInfo{
+			IsProber: true,
+		}
+		clientIP := netip.IPv6Loopback()
```

Comment:
Is there a particular reason that we use the IPv6 loopback addr here?

Reply:
It's not even used on the other side so could just be netip.Addr{} I think

Reply:
It was the first easy to reach for symbol, and happens to match what you'd generally get for local requests, and I didn't have any reason to plumb the web request data through.

Reply:
> It's not even used on the other side so could just be netip.Addr{} I think

True, I didn't want to leave a potential panic in a future change so I initialized both of the parameters.